packages/asset-swapper/src/utils/market_operation_utils/compound_ctoken_cache.ts

```diff
@@ -0,0 +1,84 @@
+import { logUtils } from '@0x/utils';
+import axios from 'axios';
+
+import { constants } from '../../constants';
+
+export interface CToken {
+    symbol: string;
+    tokenAddress: string;
+    underlyingAddress: string;
+    underlyingSymbol: string;
```

Comment:
are the symbols used?

Reply:
They are not used for anything, kept them around to simplify debugging but can remove them 👍